### ISSUE TITLE
[FileReader] Upon restart, it ingests already ingested logs. 

### ISSUE THREAD

**Issue by rockb1017** [open-telemetry/opentelemetry-collector-contrib#2947]
**Describe the bug**
When I upgrade otel helm chart, it recreates collector pod, then I have duplicate of logs. 
So the file checkpointing is not working properly.

**Comment by tigrannajaryan** [comment]
@rockb1017 Checkpoint saving is not implemented yet, see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2287. Also, show your config to make reproduction easier.

@djaglowski is it expected that for now on restart we will re-read already read files or that we will completely skip them?

**Comment by djaglowski** [comment]
As Tigran mentioned, checkpoints depend on #2287, which is in progress.

In the absence of a checkpoint, the behavior is controlled by the `start_at` configuration parameter, documented [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver#configuration).

**Comment by djaglowski** [comment]
I believe this is resolved, now that the [file_storage extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage) is available. Note that a `file_storage` extension needs to be configured in order for this to work.
